auacm/angular/src/app/edit-problem/edit-problem.component.html

```diff
@@ -0,0 +1,130 @@
+            placeholder="Some ridiculous plot..."></textarea>
+        </div>
+
+        <!-- Input Description -->
+        <div class="form-group">
+          <label for="inputDescription">Input Description</label>
+          <textarea class="form-control" rows="4"
+            id="inputDescription" placeholder="n can be between -4 and 10,000"
+            [(ngModel)]="problem.inputDesc" name="problemInput"></textarea>
+        </div>
+
+        <!-- Output Description -->
+        <div class="form-group">
+          <label  for="outputDescription">Output Description</label>
+          <textarea class="form-control" rows="4"
+            id="outputDescription" placeholder="Answer should be to 42 decimal places"
+            [(ngModel)]="problem.outputDesc" name="problemOutput"></textarea>
+        </div>
+
+        <!-- Sample Input/Output -->
+        <div *ngIf="showCaseList">
+          <div class="form-group" *ngFor="let case of problem.sampleCases; let i = index">
+            <label>Sample Case {{ i + 1 }}</label>
+            <div class="form-inline">
+              <textarea class="form-control"
+                placeholder="Sample input"
+                ngDefaultControl [(ngModel)]="case.input" name="{{'caseInput' + i}}"
+                style="width:40%;"></textarea>
+              <textarea class="form-control"
+                placeholder="Sample output"
+                ngDefaultControl [(ngModel)]="case.output" name="{{'caseOutput' + i}}"
+                style="width:40%; margin-left: 10%"></textarea>
+            </div>
+          </div>
+        </div>
+        <input type="button" class="btn btn-info"
+          (click)="addCase()" value="Add Test Case" style="margin-bottom: 25px">
+        <input type="button" class="btn btn-default pull-right"
+          (click)="removeCase()" value="Delete" *ngIf="problem.sampleCases.length > 0">
+        <div *ngIf="addCaseError">
+          <div class="label label-danger">{{addCaseErrorMsg}}</div>
+        </div>
+
+        <!-- Judge Input/Output -->
+        <div class="form-group">
+          <div class="form-inline">
+            <label for="inputUpload">Judge Input</label>
+            <input type="file" (change)="judgeInput = $event.target.files[0]" accept=".zip">
+            <p class="help-block">All files should be in an 'in' directory and zipped</p>
+          </div>
+        </div>
+        <div class="form-group">
+          <div class="form-inline">
+            <label for="outputUpload">Judge Output</label>
+            <input type="file" (change)="judgeOutput = $event.target.files[0]" accept=".zip">
+            <p class="help-block">All files should be in an 'out' directory and zipped</p>
+          </div>
+        </div>
+
+        <!-- Judge Solution -->
+        <div class="form-group">
+          <label for="judgeSolution">Judge Solution</label>
+          <input type="file" (change)="judgeSolution = $event.target.files[0]" accept=".c,.cpp,.java,.py">
```

Comment:
The judge page allows 
>.java,.py,.c,.cpp,.c++,.go
Is there a reason we have a smaller list of accepted files here?